test_site/pages/home.rb

```diff
@@ -13,11 +13,13 @@ class TestHomePage < SitePrism::Page
   element :invisible_element, 'input.invisible'
   element :shy_element, 'input#will_become_visible'
   element :retiring_element, 'input#will_become_invisible'
+  element :removing_element, 'input#will_become_nonexistent'
   element :remove_container_with_element_btn, 'input#remove_container_with_element'
 
   # elements groups
   elements :lots_of_links, :xpath, '//td//a'
   elements :nonexistent_elements, 'input#nonexistent'
+  elements :removing_elements, 'input#will_become_nonexistent'
```

Comment:
this probably needs to be more granular (I have a PR in place which removes the "hack" that makes this work)